### DISCUSSION TITLE
MS Edge in IE Mode?

### DISCUSSION THREAD

**Discussion by alekks** [seleniumbase/SeleniumBase#751] [General]
Quick question, tried to find some documentation about it, but could not find anything.
Am I able to run Microsoft Edge in IE mode with SeleniumBase? I've spent a couple of days trying with regular Selenium but just can't figure it out, so I found SeleniumBase and it seems to make fewer headaches. :) So, my only requirement at the moment is to be able to run MS Edge in IE Mode, is that feasible?

Thanks.

**Comment by mdmintz** [comment]
Hi @alekks, you can run SeleniumBase with Edge using ``--browser=edge`` and SeleniumBase with IE using ``--browser=ie``, but Edge in IE-Mode is not currently supported, so consider using the two browsers separately when using SeleniumBase.

**Reply by alekks** [reply]
Thanks for your reply. 
Windows seem to have started pushing for switching from IE to Edge now, after last windows update I now get a wizard to choose whether to use recommended settings or not when launching IE. Can't find a way to target this dialog with SeleniumBase, and with legacy systems, Edge in IE mode is still very much a thing and will be for several years to come. 
So is running Edge in IE mode planning to be supported? 

Thanks again.

**Reply by mdmintz** [reply]
Hi @alekks, The info on https://pureinfotech.com/enable-ie-mode-microsoft-edge-chromium/ has the steps you're looking for. With SeleniumBase, you can pass a custom chromium/edge arg using ``--chromium-arg="--ARG"`` on the command line once you know what args you need to pass to enable what you want. That can be a comma-separated list if necessary.

**Reply by alekks** [reply]
Thanks for the info! Thought that I had done my google well enough, apparently not :) Cheers.
